docs/README.md

```diff
@@ -365,46 +365,49 @@ Set a custom `Host` header to work around missing DNS records:
 $ http localhost:8000 Host:example.com
 ```
 
-## HTTP method
+## HTTP methods
```

Comment:
We generate section URLs from their header text, so I’d keep the singular. (https://httpie.io/docs/cli/http-method)